README.md

```diff
@@ -1,6 +1,8 @@
 # Forest Admin in Rails
 [![Gem](https://badge.fury.io/rb/forest_liana.svg)](https://badge.fury.io/rb/forest_liana)
-[![CI status](https://travis-ci.org/ForestAdmin/forest-rails.svg?branch=devel)](https://travis-ci.org/ForestAdmin/forest-rails)
+[![CI status](https://travis-ci.org/ForestAdmin/forest-rails.svg?branch=master)](https://travis-ci.org/ForestAdmin/forest-rails)
+[![codecov](https://codecov.io/gh/ForestAdmin/forest-rails/branch/master/graph/badge.svg)](https://codecov.io/gh/ForestAdmin/forest-rails)
```

Comment:
Why do we use both Codecov and Code Climate? We should clean the Codecov badge.

Reply:
We should remove it :)